### DISCUSSION TITLE
Flask application and minio

### DISCUSSION THREAD

**Discussion by mfaridi1394** [minio/minio#15657] [Q&A] [answered]
I have two flask applications run on to different servers and I have another server, this server is NFS share server , all two flask applications use share folder on NFS server and use it , I want remove NFS server and use Minio as file sharing for these two flask applications.
What is best solution for this project? 
I don not have enough knowledge about python and I can not modify python code
I need only use Minio for solve this problem

**Comment by klauspost** [comment, marked as answer]
Use an S3 SDK in your application to access data. For example [MinIO Python SDK](https://docs.min.io/docs/python-client-quickstart-guide.html).

**Reply by mfaridi1394** [reply]
> Use an S3 SDK in your application to access data. For example [MinIO Python SDK](https://docs.min.io/docs/python-client-quickstart-guide.html).

Thanks,
Can I use MinFs for solve this problem?

**Reply by klauspost** [reply]
No. It is no longer supported. There are alternatives but none work well in my experience.

Mounting S3 as a filesystem will never work well. Especially if you have multiple reading/writing from the same bucket.

You asked for the "best solution". That is to use an S3 SDK.

**Reply by mfaridi1394** [reply]
> No. It is no longer supported. There are alternatives but none work well in my experience.
> 
> Mounting S3 as a filesystem will never work well. Especially if you have multiple reading/writing from the same bucket.
> 
> You asked for the "best solution". That is to use an S3 SDK.

Thanks another solution?

**Reply by klauspost** [reply]
You can try an FS mount, but it will likely will not work well.
